src/index.html

```diff
@@ -1,17 +1,184 @@
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="submit">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <!-- /* surname */ -->
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input type="text"
+            name="surname"
+            id="surname"
+            required
+            autocomplete="off">
+        </div>
+
+
+        <!-- /* name */ -->
+        <div class="input">
+          <label for="name">Name:</label>
+          <input type="text"
+            name="name"
+            id="name"
+            required
+            autocomplete="off">
+        </div>
+
+
+        <!-- /* age */ -->
+        <div class="input">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            required
+            min="1"
+            max="100"
+            value="12">
+        </div>
+
+
+        <!-- /* birthday */ -->
+        <div class="input">
+          <label for="birth">Full date of birth:</label>
+          <input type="date"
+            name="birth"
+            id="birth"
+            required>
+        </div>
+
+        <!-- /* accept */ -->
+        <div class="input">
+          <label for="accept">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="accept"
+            id="accept">
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <!-- /* email */ -->
+        <div class="input">
+          <label for="email">E-mail:</label>
+          <input type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required>
+        </div>
+
+        <!-- /* email */ -->
+        <div class="input">
+          <label for="password">Password:</label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="25"
+            required>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <!-- /* love cats? */ -->
+        <div class="input">
+          Do you love cats?
+          <label>
+            <input type="radio"
+              name="love_cats"
+              value="true"> Yes
```

Comment:
fix indentation, use camelCase for names

```suggestion
            <input
              type="radio"
              name="loveCats"
              value="true"
             >
              Yes
```